app/models/lionpath/lionpath_export_payload.rb

```diff
@@ -0,0 +1,83 @@
+class Lionpath::LionpathExportPayload
+  def initialize(submission)
+    @submission = submission
+  end
+
+  def json_payload
+    internal_data = {}
+    {
+      "emplid" => submission.author.psu_idn,
+      "candNbr" => submission.candidate_number,
+      "thesisTitle" => submission.title,
+      "thesisStatus" => thesis_status,
+      "embargoType" => embargo_type,
+      "embargoStartDt" => submission.released_metadata_at&.strftime("%Y%m%d"),
+      "embargoEndDt" => submission.released_for_publication_at&.strftime("%Y%m%d"),
+      "candAdvFlg" => core_committee_approved,
+      "exPymtFlg" => payment_received,
+      "libDepFlg" => federal_funding_used,
+      "grdtnFlg" => program_head_approved
+    }.each do |key, value|
+      internal_data[key] = value if value
+    end
+    { "PE_SR199_ETD_REQ": internal_data }.to_json
+  end
+
+  private
+
+    attr_accessor :submission
```

Comment:
It was mostly just oversight on my part.  I usually start with accessor while writing the code just in case.  In this case, reader is all that's needed